tutorials/pulsar_analysis.ipynb

```diff
@@ -0,0 +1,464 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that the phases are computed, we want to do a phase-resolved sky map : a map of the ON-phase events minus alpha times the OFF-phase events."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gammapy.maps import Map\n",
+    "from gammapy.cube import fill_map_counts, MapMaker\n",
+    "\n",
+    "\n",
+    "# Let's create an ON-map and an OFF-map\n",
+    "on_map = Map.create(binsz=0.1, map_type='wcs', skydir=pos_target, width=10.0)\n",
+    "off_map = Map.create(binsz=0.1, map_type='wcs', skydir=pos_target, width=10.0)\n",
+    "\n",
+    "# Loop to fill the ON- and OFF- maps\n",
+    "on_mask = np.where(np.logical_and(0.5 < phases, 0.6 > phases))[0]\n",
+    "fill_map_counts(on_map, one_obs_vela_select.select_row_subset(on_mask))\n",
+    "off_mask = np.where(np.logical_and(0.7 < phases, 1 > phases))[0]\n",
+    "fill_map_counts(off_map, one_obs_vela_select.select_row_subset(off_mask))\n",
+    "\n",
+    "# Create and fill excess map\n",
+    "excess_map = Map.create(binsz=0.1, map_type='wcs', skydir=pos_target, width=10.0)\n",
+    "excess_map.data = on_map.data - 0.1/0.3 * off_map.data\n",
+    "\n",
+    "# Plot excess map\n",
+    "excess_map.plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can smooth it\n",
+    "\n",
+    "excess_map.smooth(kernel='gauss', width=0.2 * u.deg).plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## IV : Phase-resolved spectrum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also do a phase-resolved spectrum."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gammapy.background.phase import PhaseBackgroundEstimator\n",
+    "from regions import CircleSkyRegion\n",
+    "from gammapy.utils.energy import EnergyBounds\n",
+    "from gammapy.spectrum.models import PowerLaw\n",
+    "from gammapy.spectrum import (\n",
+    "    SpectrumExtraction,\n",
+    "    SpectrumFit,\n",
+    "    SpectrumResult,\n",
+    "    models,\n",
+    "    SpectrumEnergyGroupMaker,\n",
+    "    FluxPointEstimator,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "on_region = CircleSkyRegion(pos_target, on_radius)\n",
+    "\n",
+    "# The PhaseBackgroundEstimator uses the OFF-phase in the ON-region to estimate the background\n",
+    "bkg_estimator = PhaseBackgroundEstimator(observations=obs_list_vela, on_region=on_region, on_phase=(0.5, 0.6), off_phase = (0.7, 1))\n",
+    "bkg_estimator.run()\n",
+    "bkg_estimate = bkg_estimator.result"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Again explain a bit, what is done now. (You can also refer to the spectrum extraction tutorial but at least explain briefly what is done)